mu4/cloud/internal/cloudmanager_p.h

```diff
@@ -59,12 +59,14 @@ class ApiInfo
     static constexpr const char* apiKeyHeader = "X-MS-API-KEY";
     static constexpr const char* userAgentTemplate = "MS_EDITOR/%1.%2 (%3)";
 
+    static constexpr const char* registerPage = "https://musescore.com/user/register?webview";
     static constexpr const char* loginPage = "https://musescore.com/user/auth/webview";
     static constexpr const char* loginSuccessPage = "https://musescore.com/user/auth/webview/success";
```

Comment:
It is better to write constants with an Upper case

Reply:
fixed